_posts/2025-06-12-hardware-in-the-loop-simulation-support-for-subnero-modems.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Hardware-in-the-Loop Simulation Support for Subnero Modems
+banner: images/banner-vao.jpg
+date:  2025-06-12
+categories: wnc news
+thumbnail: images/pulse-thumbnail-vao.jpg
+---
+
+We’re excited to announce hardware-in-the-loop (HIL) simulation support across our entire range of acoustic smart modems powered by UnetStack. This new capability is being launched at the OCEANS 2025 Conference in Brest, where we’ll also host live demonstrations at our booth.
+
+This capability allows any Subnero modem running UnetStack v5 to be configured to interface with a local simulation environment—enabling early-stage testing and iteration before investing in costly and time-consuming at-sea deployments.
+
+At the core of this simulation framework is the open-source [Virtual Acoustic Ocean (VAO)](https://github.com/org-arl/VirtualAcousticOcean.jl), which acts as the acoustic channel simulator, along with the [UASPv2 protocol](https://github.com/org-arl/VirtualAcousticOcean.jl/blob/main/docs/uasp2-protocol.md) for seamless communication integration. The platform supports industry-standard channel models such as Bellhop, Kraken, and Pekiris RAT, and allows users to integrate their own channel models for even greater fidelity and customization.
```

Comment:
What's a Pekiris RAT? In any case, Bellhop and Kraken support has some constraints at present due to upstream limitations in `UnderwaterAcoustics.jl`. Best not to specifically mention them at this point in time.

Reply:
Removed the mention of specific models, not sure what the RAT was, thanks for catching.